BrewMesModulesParent/common/src/main/java/com/brewmes/common/services/IMachineService.java

```diff
@@ -12,21 +12,21 @@
      * Takes a command and sends it to the desired machine
```

Comment:
The changes made in this file seems unnecessary?

Reply:
Changing a variable name to a more descriptive name is not unnecessary in my opinion